www/js/lib/uiUtil.js

```diff
@@ -32,6 +32,48 @@ if (webpMachine) {
 }
 
 define(rqDef, function(settingsStore) {
+
+    /**
+     * Displays a Bootstrap alert or confirm dialog box depending on the options provided
+     * 
+     * @param {String} message The alert message to display in the body of the modal 
+     * @param {String} label The modal's label or title which appears in the header (optional, Default = "Confirmation" or "Message")
+     * @param {Boolean} isConfirm If true, the modal will be a confirm dialog box, otherwise it will be a simple alert message 
+     * @param {String} declineConfirmLabel The text to display on the decline confirmation button (optional, Default = "Cancel") 
+     * @param {String} approveConfirmLabel  The text to display on the approve confirmation button (optional, Default = "Confirm")
+     * @param {String} closeMessageLabel  The text to display on the close alert message button (optional, Default = "Okay")
+     * @returns {Promise<Boolean>} A promise which resolves to true if the user clicked Confirm, false if the user clicked Cancel/Okay, backdrop or the cross(x) button
+     */
+    function systemAlert(message, label, isConfirm, declineConfirmLabel, approveConfirmLabel, closeMessageLabel) {
+        declineConfirmLabel = declineConfirmLabel || "Cancel";
+        approveConfirmLabel = approveConfirmLabel || "Confirm";
+        closeMessageLabel = closeMessageLabel || "Okay";
+        label = label || (isConfirm ? "Confirmation" : "Message");
+        return new Promise(function (resolve, reject) {
+            if (!message) reject("Missing body message");
+            // Set the text to the modal and it's buttons
+            document.getElementById("approveConfirm").textContent = approveConfirmLabel;
+            document.getElementById("declineConfirm").textContent = declineConfirmLabel;
+            document.getElementById("closeMessage").textContent = closeMessageLabel;
+            document.getElementById("modalLabel").textContent = label;
+            document.getElementById("modalText").textContent = message;
```

Comment:
@mossroy Although we have #824 , it strikes me that this spot is precisely a case where we *do* want the capability to insert HTML. One of the big wins of using Bootstrap instead of alert() and confirm() is that we can style messages with HTML (use emphasis, colours, spacing, bold...).. Given that we ourselves set the message strings, it seems to me that there is zero risk here (this is not content from the ZIM or inserted by users). For at least the message and the label, but possibly also the buttons, it would be great to have the ability to typeset/compose longer messages or dialogue boxes nicely.

Reply:
I agree with you.
We might keep `.innerHTML` for the `modalText`

Reply:
Yes, this makes sense. I'll add a commit for this.

Reply:
Yeah, that makes sense, my bad. Will fix it.